10/umbraco-forms/developer/ajaxforms.md

```diff
@@ -409,7 +409,7 @@ Firstly, with server-to-server integrations you will want to disable the antifor
 
 This is done by setting the `Umbraco:Forms:Security:EnableAntiForgeryTokenForFormsApi` configuration key to a value of `false`.
 
-You should then configure an API key `Umbraco:Forms:Security:FormsApiKey`.  This can be any string value, but shouldn't be easily guessable by a brute force attack.
+You should then configure an API key `Umbraco:Forms:Security:FormsApiKey`.  The string value can be any, but it should not be susceptible to a brute force attack.
```

Comment:
Not sure if this should be "any value" or "anything", like I've suggestion.
But I felt something was missing from the sentence. What do you think?

```suggestion
You should then configure an API key `Umbraco:Forms:Security:FormsApiKey`. The string value can be anything, but it should not be susceptible to a brute force attack.
```

Reply:
@sofietoft I had exactly the same doubt 🙈. Just updated the PR with your suggestion 😄